scripts/hail_batch/ld_prune/hgdp_1kg_ld_prune.py

```diff
@@ -22,6 +22,9 @@ def query(output):
     # reproduce gnomAD genotype filtering
     mt = annotate_adj(mt)
     mt = mt.filter_entries(mt.adj)
+    # Filter to common variants
+    mt = hl.variant_qc(mt)
+    mt = mt.filter_rows(mt.variant_qc.AF[1] > 0.05)
```

Comment:
Should this filter step happen after `biallelic_mt`, to make sure there are two `AF` entries?

Or maybe this could even be combined with the `filter_rows` operation below?

Reply:
I combined filtering for biallelic and common variants. Let me know what you think :)